pages/k8s/monitoring.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Monitoring | Canonical Distribution of Kubernetes&reg;"
+keywords: juju, monitor, grafana, prometheus
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: monitoring.html
+layout: base
+toc: False
+summary: This page is currently a work in progress. For existing documentation, please visit <a href="https://kubernetes.io/docs/getting-started-guides/ubuntu/"> https://kubernetes.io/docs/getting-started-guides/ubuntu/ </a>
+---
+
+# Monitoring
+
+The **Canonical Distribution of Kubernetes**<sup>&reg;</sup> includes the
+standard **Kubernetes** dashboard for monitoring your cluster. However, it is
+often advisable to have a monitoring solution which will run whether the
+cluster itself is running or not. It may also be useful to integrate monitoring
+into existing setups.
+
+**Prometheus** is the recommended way to monitor your deployment -
+instructions are provided below. There are also instructions for setting up
+other monitoring solutions, or connecting to existing monitoring setups.
+
+## Monitoring with Prometheus
+
+The recommended way to monitor your cluster is to use a combination of
+**Prometheus**, **Grafana** and **Telegraf**. The fastest, easiest way to
+install and configure this is via **conjure-up** when installing the
+**Canonical Distribution of Kubernetes**<sup>&reg;</sup>, by selecting the box
+next to  Prometheus from the `Add-on` menu. You can then log in to the dashboard
+as [described below](#retrieve-credentials-and-login). See the [quickstart
+guide][quickstart] for more details on installing **CDK** with **conjure-up**.
+
+If you have already installed your cluster, you will be able to add and
+configure  the extra applications using **Juju** as described here:
+
+### Install the required applications
+
+The following commands will add the required applications:
+
+```bash
+juju deploy grafana --series=xenial
+juju deploy prometheus  --series=xenial
+juju deploy telegraf  --series=xenial
```

Comment:
@evilnick you getting paid by the `<space>` nowadays?

Reply:
it's part of a conspiracy to fill up the internet